src/bluesky/utils/__init__.py

```diff
@@ -629,8 +629,8 @@ def snake_cyclers(cyclers, snake_booleans):
         lengths.append(len(c))
     total_length = np.prod(lengths)
     for i, (c, snake) in enumerate(zip(cyclers, snake_booleans)):
-        num_tiles = np.prod(lengths[:i])
```

Comment:
Somehow despite the commit message I must have messed up the rebase and changed it to the wrong one 🤦🏻